docs/API.md

```diff
@@ -1671,23 +1671,24 @@ hooks in the plugins bundled with Flot.
     The resize hook is used to be notified after the plot was resized.
 
  - findNearbyItems   [phase 7]
-     ```function (canvasX, canvasY, seriesFilter, distance)```
+     ```function (canvasX, canvasY, series, radius, computeDistance, items)```
     
     The findNearbyItems hook is used to extend the default behavior of
-    the flot-provided 'findNearbyItems' function. The hover plugin is
-    an example of a plugin that leverages this method to return items 
-    that will be highlighted when hovered over. So, as an example, a new
-    plugin might provide this hook to allow the hover plugin to highlight
-    points that the flot-provided findNearbyItems method would have 
-    otherwise not found.
+    the flot-provided 'findNearbyItems' function. The hover plugin will take
```

Comment:
I don't really like the wording of this - "will take advantage", "new capability", etc, that sort of phrasing will become out of date pretty quickly.